woocommerce/woocommerce-android
# Change color on payment provider icon to support dark mode

Closes: #6932 


### Description


Changes the color of the ic_payment_provider from white to `@color/color_on_surface` to add support for dark mode

context: [p5T066-3oX-p2](p5T066-3oX-p2#comment-12929)

### Testing instructions


1. Enable dark mode on your device
2. Access a store in the US with both WCPay and Stripe enabled
3. go to menu
4. go to IPP settings
5. Select the payment provider and click on confirm payment method
6. The app will take you back to the IPP settings screen. Make sure the icon on the Change Payment Provider row shows in white

### Images/gif


<img width="351" alt="Screen Shot 2022-07-14 at 8 13 51 AM" src="https://user-images.githubusercontent.com/30724184/178971577-d6bba5ef-fe95-4bea-8ec8-0d76d33cbff9.png">
